File: app/src/main/java/android/itacademy/by/dz6/ImageLoaderUtil.java
package android.itacademy.by.dz6;

import android.widget.ImageView;

import com.squareup.picasso.Picasso;

public class ImageLoaderUtil {
    public static void loadImage(ImageView imageView,String url){
        Picasso.get().load(url).into(imageView);
    }
    public static void cancelLoadImage(ImageView imageView){
        Picasso.get().cancelRequest(imageView);
    }
}

File: app/src/main/java/android/itacademy/by/dz6/imageloader/ImageLoaderUtil.kt
package android.itacademy.by.dz6.imageloader

import android.widget.ImageView

import com.squareup.picasso.Picasso

object ImageLoaderUtil {
    fun loadImage(imageView: ImageView, url: String) {
        Picasso.get().load(url).into(imageView)
    }
}